src/utils.ts

```diff
@@ -1,21 +1,23 @@
-import { workspace, Disposable, DocumentSelector } from 'vscode';
+import { workspace, Disposable, DocumentSelector, Uri } from 'vscode';
 import { PrettierVSCodeConfig } from './types.d';
 
-let currentRootPath: string = workspace.rootPath;
+let currentRootPath: string = workspace.rootPath!;
```

Comment:
General question, what is the `!;` at the end, and what does it do and why do we need it?

> It's probably my lack of TypeScript knowledge, but can't easily find what it is.

Reply:
It means 'I, as a coder, assert: it **IS** defined'

Reply:
oh, til!